packages/js/src/Client.ts

```diff
@@ -323,6 +324,34 @@ export default class Client {
     return authorization
   }
 
+  getAuthorizationProof(
```

Comment:
Won't this throw if there isn't a Solana proof. I feel like we should just return null. That way the app will be forced to account for this function getting depreciated and just returning null. WDYT?

Reply:
Good point 👍